langstroth/gradlew

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
```

Comment:
The gradle wrapper you generated needs a couple more files, can you run ~gradle wrapper~ and then add the `./gradle` folder generated? I can add it, but we would get a newer version of gradle, which can be good/bad depending on what we want.